docs/vector_store.ipynb

```diff
@@ -349,6 +349,48 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Initialize Vector Store with documents from any source\n",
```

Comment:
We need to describe the difference between the default above the this option. Let's remove "from any source" because that is potentially confusing to users. We should also add a note that this is a great way to get started but the default way is most likely how an app will use the integration since we won't accidentally add the same documents.